bedrock/products/templates/products/vpn/landing.html

```diff
@@ -351,4 +351,5 @@ <h3>{{ ftl('vpn-landing-faq-download-heading') }}</h3>
   {% endif %}
 
   {{ js_bundle('mozilla-vpn-landing') }}
+  {{ js_bundle('track_begincheckout') }}
```

Comment:
Question (non-blocking): should we try and avoid using the word track in filenames? That seems like a common word for extensions to block loading? I'm guessing they will likely block GA anyways, but we should try and prevent our own scripts from being blocked at least?

Reply:
Yeah, I had the same thought and then decided I was being paranoid and opted for readability for devs. But if that makes two of us I should change it. Event? Log?

Reply:
Went with "data"